R/get_summary.R

```diff
@@ -30,8 +30,8 @@ get_summary <- function(mydir, para, vec, name, profilemodels, profilesummary) {
     "likelihood" = sapply(sapply(outputs, "[[", "likelihoods_used", simplify = FALSE), "[", 1, 1),
     "gradient" = sapply(outputs, "[[", "maximum_gradient_component"),
     "SB0" = sapply(quants, "[[", "SSB_Virgin", "Value"),
-    "SBfinal" = sapply(quants, "[[", paste0("SSB_", outputs[[1]]$endyr), "Value"),
-    "Deplfinal" = sapply(quants, "[[", paste0("Bratio_", outputs[[1]]$endyr), "Value"),
+    "SBfinal" = sapply(quants, "[[", paste0("SSB_", outputs[[1]]$endyr + 1), "Value"),
```

Comment:
This stuff here and below is because SS3 calculates beginning of the year biomass right? Also, would there ever be a situation where this would fail? Like the model is terminated in endyr and there are no values for endyr+1?

Reply:
The only way that I think this could break would be if someone set the max year for sdreport output to be a specific year that was either the final data year or before. I don't think r4ss has an object associated with this reported in by `SS_output`. If we think this is important to check I could add a line to read the starter file to check this setting.

Reply:
If the goal of this package is to support PFMC groundfish assessments, then all users should have this output. I think checking stuff like that could go into a ToR checker function as discussed in https://github.com/pfmc-assessments/PEPtools/issues/13.

Reply:
Agreed. I have seen users accidentally have the starter input set incorrectly leading to issues but I think users are less likely to now accidentally encounter it since SS3 was revised to populate the starter.ss_new file with a value of -2 that by default includes endyr+Nforecastyrs.